examples/tensorflow/tf1/README.md

```diff
@@ -0,0 +1,55 @@
+## MLflow with TensorFlow 1.X
+
+In this example, we train a `tf.estimator.DNNRegressor` on the TensorFlow [Boston Housing dataset](https://www.cs.toronto.edu/~delve/data/boston/bostonDetail.html).
+With MLflow, we use `mlflow.tensorflow.autolog()` along with `mlflow.log_param()` and `mlflow.log_metric()` in order to capture data about how we trained the target model.
+We also use MLflow in order to generically save the model as a `mlflow.pyfunc`, then load it back to predict on pandas DataFrames.
+
+### Code related to MLflow:
+* [`mlflow.tensorflow.autolog()`](https://www.mlflow.org/docs/latest/tracking.html#automatic-logging-from-tensorflow-and-keras-experimental):
+This is an experimental api that logs ML model artifacts and TensorBoard metrics created by the `tf.estimator` we are using.
+The TensorBoard metrics are logged during training of the model. By default, MLflow autologs every 100 steps.
+The ML model artifact creation is handled during the call to `tf.estimator.export_saved_model()`.
+
+* [`mlflow.log_param()`](https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.log_param):
+This manually logs a parameter that autolog doesn't cover. In this example, we log the `hidden_units` parameter.
+
+* [`mlflow.log_metric()`](https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.log_metric):
```

Comment:
I think we can delete this since autolog should log some metrics

Reply:
So do we want to leave these examples to be centered around autologging and leave `log_metric` and `log_param` to frameworks without autologging?

Reply:
Yeah I think that would be best (in an ideal world autologging would capture all the important metrics/params)